sci-biology/mmseqs2/mmseqs2-13.ebuild

```diff
@@ -0,0 +1,48 @@
+# Copyright 2021-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit bash-completion-r1 cmake
+
+DESCRIPTION="Fast and sensitive sequence search and clustering"
+HOMEPAGE="https://github.com/soedinglab/MMseqs2"
+MY_PN="MMseqs2"
+SUB_PV="45111" # this is a hex value and must be updated with each release
+MY_PV="${PV}-${SUB_PV}"
+
+SRC_URI="https://github.com/soedinglab/${MY_PN}/archive/refs/tags/${MY_PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64"
+
+# This package only builds correctly native and does not respect
+# CPU flags, despite a subset of them being defined as cmake flags.
```

Comment:
If we can't force it to respect the cpu flags we set, can we force it to build generic (i.e. with all cpu flags disabled)?

Otherwise we might want to add `RESTRICT="bindist", since binary packages of this likely won't work.

There is a similar issue in[ `sci-biology/mrbayes`](https://github.com/gentoo/gentoo/blob/master/sci-biology/mrbayes/mrbayes-3.2.7.ebuild) that caused me a bit of a headache a couple of weeks back. Current workaround is to force clear `SIMD_FLAGS` and `CPUEXT_FLAGS`, something similar might work here as well.